src/OSPSuite.Core/Services/IDialogCreator.cs

```diff
@@ -12,7 +12,7 @@ public enum ViewResult
 
    public interface IDialogCreator
    {
-      void MessageBoxError(string message);
+      void MessageBoxError(string message, bool hyperlink = false);
```

Comment:
if you need in the code to differentiate usage of hyperlink or not, I would search the message for href  or sthg like this?

Reply:
I like the suggestion, U had not thought of that. I will adjust it accordingly then.